.changeset/2863.md

```diff
@@ -0,0 +1,29 @@
+---
+'@backstage/core-api': minor
```

Comment:
This isn't actually a breaking change right, could make it patch?

Reply:
We are changing/adding functionality of APIs in a backwards compatible manner. So also a minor change could be reasonable, right? 
```
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
```

Reply:
While on v0, semver allows breaking changes in minor versions instead of major. Afaik it's not really clear from the spec where backwards compatible changes should then go, but we likely end up putting those in patch releases.

Reply:
Ah, I see! Thanks